man/detect_nonwear_clipping.Rd

```diff
@@ -0,0 +1,76 @@
+\name{detect_nonwear_clipping}
+\alias{detect_nonwear_clipping}
+\title{
+Detect non-wear and clipping time in the raw accelerometer data
+}
+\description{
+Detects periods in which the accelerometer has not been worn or the
+accelerometer signal was stuck very close to the dynamic range limit
+of the accelerometer.
+}
+\usage{
+detect_nonwear_clipping(data = c(), windowsizes = c(5, 900, 3600), sf = 100,
+                        clipthres = 7.5, sdcriter = 0.013, racriter = 0.05,
+                        nonwear_approach = "old",
+                        params_rawdata = c())
+}
+\arguments{
+  \item{data}{
+  Matrix with raw accelerometer data for X, Y, and Z axes.
+}
+ \item{windowsizes}{
+  Numeric vector of length three, with short, long epoch and window size in seconds.
+}
+ \item{sf}{
+  Sample frequency in Hertz.
+}
+ \item{clipthres}{
+  Threschold to detect clipping in _g_ units. Usually 0.5 _g_ below the 
+  dynamic range of the accelerometer.
+}
+ \item{sdcriter}{
+  Criteria to define non-wear time, defined as the estimated noise
+  measured in the raw accelerometer data.
+}
+ \item{racriter}{
+  Absolute criteria below which the absolute range of the accelerations
+  should be to define non-wear time.
+}
+ \item{nonwear_approach}{
+  Whether to use the traditional version of the non-wear detection algorithm
+  (nonwear_approach = "old", default) or the new version (nonwear_approach = "new). 
+  The old version would use the longsize window (windowsizes[3], one hour as default)
+  to check the conditions for nonwear identification and would flag as nonwear the
+  mediumsize window (windowsizes[2], 15 min as default) in the middle. The new 
+  version differs in which it would flag as nonwear the full longsize window.
```

Comment:
Add also:
"For the old method the longsize window is centered in the centre of the mediumsize window, while in the new method the longsizewindow is aligned with its left edge to the left edge of the mediumsize window."